src/Symfony/Component/FeatureFlag/FeatureChecker.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureFlag;
+
+use Symfony\Component\FeatureFlag\Exception\FeatureNotFoundException;
+
+final class FeatureChecker implements FeatureCheckerInterface
+{
+    private array $cache = [];
+
+    public function __construct(
+        private readonly FeatureRegistryInterface $featureRegistry,
+        private readonly mixed $default = false,
+    ) {
+    }
+
+    public function isEnabled(string $featureName, mixed $expectedValue = true): bool
+    {
+        return $this->getValue($featureName) === $expectedValue;
+    }
+
+    public function isDisabled(string $featureName, mixed $expectedValue = true): bool
```

Comment:
How about including this method in the interface?

Reply:
If isDisabled() returns !isEnabled everytime, i’m not sure this is really necessary.. wdyt ?

Reply:
Thinking about the full stack, we'll almost always use the interface to inject the service. Without defining the method in the interface, it is not visible.

Reply:
My bad, I forgot it.